### PR TITLE
Added cluster-health check instructions

### PR DESCRIPTION
**Pull request by rkatti**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Remember to delete this note before submitting your pull request.
>
> For pull requests on 1.18 Features: set Milestone to 1.18 and Base Branch to dev-1.18
>
> For pull requests on Chinese localization, set Base Branch to release-1.16
> Feel free to ask questions in #kubernetes-docs-zh
>
> For pull requests on Korean Localization: set Base Branch to dev-1.16-ko.\<latest team milestone>
>
> If you need Help on editing and submitting pull requests, visit:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> If you need Help on choosing which branch to use, visit:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

**Comment by rkatti** [comment]
Have signed CLA

File: content/en/docs/tasks/debug-application-cluster/debug-cluster.md [hunk 1]
```diff
@@ -29,6 +29,13 @@ kubectl get nodes
 
 And verify that all of the nodes you expect to see are present and that they are all in the `Ready` state.
 
+Also, you can run below command to get detailed information of overall cluster health.
```
Comment (sftim) [inline content/en/docs/tasks/debug-application-cluster/debug-cluster.md]:
```suggestion
To get detailed information about the overall health of your cluster, you can run:
```
?

@rkatti, what do you think?

File: content/en/docs/tasks/debug-application-cluster/debug-cluster.md [hunk 2]
```diff
@@ -29,6 +29,13 @@ kubectl get nodes
...
+
+```shell
+
```
Comment (sftim) [inline content/en/docs/tasks/debug-application-cluster/debug-cluster.md]: This blank line doesn't seem to be needed here.

**Comment by steveperry-53** [comment]
Agree with suggestion made by @sftim. Otherwise
/lgtm

**Comment by kbhawkey** [comment]
Ping @rkatti . Would you have time to address the comments?

**Comment by makoscafee** [comment]
ping @rkatti  again if you have time to address review feedback

File: content/en/docs/tasks/debug-application-cluster/debug-cluster.md (hunk 1 above)
Comment (rkatti) [reply content/en/docs/tasks/debug-application-cluster/debug-cluster.md]: @sftim Looks good

File: content/en/docs/tasks/debug-application-cluster/debug-cluster.md (hunk 2 above)
Comment (rkatti) [reply content/en/docs/tasks/debug-application-cluster/debug-cluster.md]: Agree @sftim

**Comment by rkatti** [comment]
ping @makoscafee ... Fyi have updated the PR

File: content/en/docs/tasks/debug-application-cluster/debug-cluster.md (hunk 2 above)
Comment (makoscafee) [reply content/en/docs/tasks/debug-application-cluster/debug-cluster.md]: @rikatz you forgot to address this review comment!

Comment (rikatz) [reply content/en/docs/tasks/debug-application-cluster/debug-cluster.md]: I think you mean @rkatti :)

Comment (rkatti) [reply content/en/docs/tasks/debug-application-cluster/debug-cluster.md]: @sftim @makoscafee have addressed the comments. Pls, review.

File: content/en/docs/tasks/debug-application-cluster/debug-cluster.md [hunk 3]
```diff
@@ -29,6 +29,11 @@ kubectl get nodes
 
 And verify that all of the nodes you expect to see are present and that they are all in the `Ready` state.
 
+To get detailed information about the overall health of your cluster, you can run:.
```
Comment (sftim) [inline content/en/docs/tasks/debug-application-cluster/debug-cluster.md]:
```suggestion
To get detailed information about the overall health of your cluster, you can run:
```

Comment (rkatti) [reply content/en/docs/tasks/debug-application-cluster/debug-cluster.md]: Done @sftim

**Pull request merged**